sensorprocess/sensorprocess.go

```diff
@@ -18,7 +18,7 @@ import (
 // Config holds config needed throughout the process of adding a sensor reading to the cartofacade.
 type Config struct {
 	CartoFacade       cartofacade.Interface
-	Lidar             sensors.TimedSensor
+	Lidar             sensors.TimedLidarSensor
```

Comment:
Because the title of this PR is misleading - Esha and I had originally split it into 5 PRs, but while working through them I realized that they could/needed to be condensed. Since IMU sensor process is what feeds data through the capi into cartographer, this will be part of the next PR. I will go ahead and rename this one.

Reply:
I might leave a note/todo if you are worried about keeping tracking of all the changes

Reply:
Feel free to resolve this thread either way